nextflow_schema.json

```diff
@@ -99,6 +99,78 @@
             },
             "fa_icon": "fas fa-file-signature"
         },
+        "annotation_prokka": {
+            "title": "Annotation: PROKKA",
+            "type": "object",
+            "description": "These parameters influence the annotation algorithm used by PROKKA.",
+            "default": "",
+            "properties": {
+                "annotation_prokka_metagenome": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Improves the gene prediction when the input genomes are highly fragmented.",
+                    "help_text": "Improves the gene prediction when the input provided consists of metagenomes with highly fragmented genomes.\n\n> Modifies tool parameter(s):\n> - PROKKA: `--metagenome`",
+                    "fa_icon": "fas fa-braille"
+                },
+                "annotation_prokka_rawproduct": {
+                    "type": "boolean",
+                    "description": "Suppresses the default clean-up of the gene annotations.",
+                    "help_text": "By default, annotation in PROKAA is done by alignment to other proteins in its database, or the databases the user provides via the tools `--proteins` flag. The resulting annotations are then cleaned up to standardise them to Genbank/ENA conventions.\n'Vague names' are set to 'hypothetical proteins', 'possible/probable/predicted' are set to 'putative' and 'EC/CPG and locus tag ids' are removed. For info. please check PROKKA [documentation](https://github.com/tseemann/prokka).\nThis flag suppresses this default behavior of PROKKA.\n\n> Modifies tool parameter(s):\n> - PROKKA: `--rawproduct`",
+                    "fa_icon": "fab fa-product-hunt"
+                },
+                "annotation_prokka_kingdom": {
+                    "type": "string",
+                    "default": "Bacteria",
+                    "fa_icon": "fab fa-accusoft",
+                    "description": "Specifies the kingdom that the input represents.",
+                    "help_text": "Specifies the kingdom that the input represents. Attention: PROKKA cannot annotate Eukaryotes.\n\n> Modifies tool parameter(s):\n> - PROKKA: `--kingdom`",
+                    "enum": ["Archaea", "Bacteria", "Mitochondria", "Viruses"]
+                },
+                "annotation_prokka_gcode": {
+                    "type": "integer",
+                    "default": 0,
+                    "minimum": 0,
+                    "maximum": 25,
+                    "description": "Specifies the translation table used to annotate the sequences.",
+                    "help_text": "Specifies the translation table used to annotate the sequences. All possible genetic codes (1-25) used for gene annotation can be found [here](https://en.wikipedia.org/wiki/List_of_genetic_codes). This flag is required if the flag `--kingdom` is assigned. \n\n> Modifies tool parameter(s):\n> - PROKKA: `--gcode`",
```

Comment:
In the other annotation tools the translation table is set to 11 by default and the value is supposed to be between 1-25. Why is it 0 here (minimum and default)?

Reply:
its default here, but you are correct, ill change it to 11 so that all teh tools have the same parametrs